### PR TITLE
Add apigroup to the data source of pvc description

### LINKED ISSUE #82463: kubectl describe pvc should also show APIGoup of DataSource
**What happened**:

https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/kubectl/pkg/describe/versioned#L1551

Miss APIGroup of DataSource.

**What you expected to happen**:

kubectl describe pvc should also describe the apigroup

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

/kind feature
/kind bug

**What this PR does / why we need it**:

We missed it in https://github.com/kubernetes/kubernetes/pull/76463

**Which issue(s) this PR fixes**:

Fixes #82463

**Special notes for your reviewer**:

/cc @msau42 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go [hunk 1]
```diff
@@ -1549,6 +1549,9 @@ func describePersistentVolumeClaim(pvc *corev1.PersistentVolumeClaim, events *co
 		}
 		if pvc.Spec.DataSource != nil {
 			w.Write(LEVEL_0, "DataSource:\n")
+			if pvc.Spec.DataSource.APIGroup != nil {
+				w.Write(LEVEL_1, "APIGroup:\t%v\n", *pvc.Spec.DataSource.APIGroup)
+			}
 			w.Write(LEVEL_1, "Name:\t%v\n", pvc.Spec.DataSource.Name)
 			w.Write(LEVEL_1, "Kind:\t%v\n", pvc.Spec.DataSource.Kind)
```
[the remark this answers is not in the document]
Comment (cwdsuzhou) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: That's better, thanks!

done

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

File: staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go [hunk 2]
```diff
@@ -1551,6 +1551,9 @@ func describePersistentVolumeClaim(pvc *corev1.PersistentVolumeClaim, events *co
 			w.Write(LEVEL_0, "DataSource:\n")
 			w.Write(LEVEL_1, "Name:\t%v\n", pvc.Spec.DataSource.Name)
 			w.Write(LEVEL_1, "Kind:\t%v\n", pvc.Spec.DataSource.Kind)
+			if pvc.Spec.DataSource.APIGroup != nil {
```
Comment (msau42) [inline staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: Should the order be large->small, or small->large? ie

* apigroup, kind, name
* name, kind, apigroup

It looks like other places do the first, so we should try to be consistent.

**Review by msau42 — COMMENTED** [review]
/lgtm cancel
One comment about ordering

File: staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go (hunk 2 above)
Comment (liggitt) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: I'd expect group, kind, name

Comment (cwdsuzhou) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: > I'd expect group, kind, name

I also expect this. wdyt? @msau42

Comment (msau42) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: sgtm

Comment (cwdsuzhou) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: done

**Pull request merged**